java-commons/src/main/java/ru/progrm_jarvis/javacommons/lazy/Lazy.java

```diff
@@ -33,6 +33,15 @@
      */
     boolean isInitialized();
 
+    /**
+     * Gets the value wrapped by this lazy if it is already initialized, otherwise {@code null}
+     *
+     * @return the value wrapped by this lazy if it is already initialized, otherwise {@code null}
+     */
+    default T getInitializedOrNull() {
+        return isInitialized() ? get() : null;
```

Comment:
Simple implementation does violate contract for some implementations (especially, concurrent)

Reply:
Manually changed branch to `lazy-improvements` for futher work